affected_purls:
- pkg:pypi/zope@4.0
- pkg:pypi/zope@4.1
- pkg:pypi/zope@4.1.1
- pkg:pypi/zope@4.1.2
- pkg:pypi/zope@4.1.3
- pkg:pypi/zope@4.2
- pkg:pypi/zope@4.2.1
- pkg:pypi/zope@4.3
- pkg:pypi/zope@4.4
- pkg:pypi/zope@4.4.1
- pkg:pypi/zope@4.4.2
- pkg:pypi/zope@4.4.3
- pkg:pypi/zope@4.4.4
- pkg:pypi/zope@4.5
- pkg:pypi/zope@4.5.1
- pkg:pypi/zope@4.5.2
- pkg:pypi/zope@4.5.3
- pkg:pypi/zope@4.5.4
- pkg:pypi/zope@4.5.5
- pkg:pypi/zope@4.6
- pkg:pypi/zope@4.6.1
- pkg:pypi/zope@4.6.2
- pkg:pypi/zope@5.0
- pkg:pypi/zope@5.1
- pkg:pypi/zope@5.1.1
- pkg:pypi/zope@5.1.2
- pkg:pypi/zope@5.2
- pkg:pypi/zope@5.2.1
aliases:
- CVE-2021-32811
- GHSA-g4gq-j4p2-j8fr
fixed_purl:
- pkg:pypi/zope@4.6.3
- pkg:pypi/zope@5.3
references:
- id: 161150
  reference_id: ''
  url: https://github.com/zopefoundation/Zope/commit/f72a18dda8e9bf2aedb46168761668464a4be988
- id: 161149
  reference_id: CVE-2021-32811
  url: https://nvd.nist.gov/vuln/detail/CVE-2021-32811
- id: 161151
  reference_id: GHSA-g4gq-j4p2-j8fr
  url: https://github.com/advisories/GHSA-g4gq-j4p2-j8fr
- id: 161148
  reference_id: GHSA-g4gq-j4p2-j8fr
  url: https://github.com/zopefoundation/Zope/security/advisories/GHSA-g4gq-j4p2-j8fr
- id: 161126
  reference_id: GHSA-qcx9-j53g-ccgf
  url: https://github.com/zopefoundation/AccessControl/security/advisories/GHSA-qcx9-j53g-ccgf
severities:
- id: 23746
  reference_id: 161151
  scoring_elements: ''
  scoring_system: cvssv3.1_qr
  value: HIGH
- id: 23745
  reference_id: 161148
  scoring_elements: ''
  scoring_system: cvssv3.1_qr
  value: HIGH
- id: 23738
  reference_id: 161126
  scoring_elements: ''
  scoring_system: cvssv3.1_qr
  value: MODERATE
summary: Remote Code Execution via Script (Python) objects under Python 3
vulnerability_id: VCID-84uj-k1yc-aaad
weaknesses: []
